### PR TITLE
Update kubeseal template

### LINKED ISSUE #603: ark get kubeseal has outdated version
kubeseal library source is outdated. running ark get kubeseal downloads and older version 0.14.

image

using versioned fails

image

## Expected Behaviour

to install kubeseal latest

## Current Behaviour

installs an older version

## Are you a GitHub Sponsor (Yes/No?)

Check at https://github.com/sponsors/alexellis

- [ ] Yes
- [x] No

## Possible Solution

the file format seems to be wrong.

- https://github.com/bitnami-labs/sealed-secrets/releases/download/v0.14.1/kubeseal-darwin-amd64

and should be

- https://github.com/bitnami-labs/sealed-secrets/releases/download/v0.17.2/kubeseal-0.17.2-darwin-amd64.tar.gz

there is a version number embedded in the filename

## Steps to Reproduce (for bugs)
1. ark get kubeseal@v0.17.2

## Context

N/A

## Your Environment

N/A

```
kubectl version
```

* Operating System and version (e.g. Linux, Windows, MacOS):

N/A

* What arkade version is this?

```
arkade version
Version: 0.8.0
```

### PR DESCRIPTION
**Pull request by yankeexe**
Signed-off-by: Yankee Maharjan <yankee.exe@gmail.com>

## Description

Closes #603 

`kubeseal` updated their release format: 
- previous: binary
- now: tar.gz

Update the template to get the latest distribution format.

## Motivation and Context

- [x] I have raised an issue to propose this change, which has been given a label of `design/approved` by a maintainer ([required](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md))

## How Has This Been Tested?

```bash
go run . get kubeseal
```

<details>
<summary><strong>Demo</strong></summary>

</details>

<details>
<summary><strong>Test on RPi</strong></summary>

</details>

## Are you a GitHub Sponsor (Yes/No?)

- [ ] Yes
- [x] No

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Documentation

- [x] I have updated the list of tools in README.md if (required) with `./arkade get -o markdown`
- [ ] I have updated the list of apps in README.md if (required) with `./arkade install --help`

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

- [x] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

File: pkg/get/tools.go [hunk 1]
```diff
@@ -348,28 +348,27 @@ https://storage.googleapis.com/kubernetes-release/release/{{.Version}}/bin/{{$os
 			Owner:       "bitnami-labs",
 			Repo:        "sealed-secrets",
 			Name:        "kubeseal",
-			Version:     "v0.14.1",
+			Version:     "v0.17.2",
```
Comment (alexellis) [inline pkg/get/tools.go:351]: We should remove this version so it can be auto-detected. Can you make that change and test again please?

**Review by alexellis — CHANGES_REQUESTED** [review]
Requesting a change.

**Comment by yankeexe** [comment]
@alexellis  Updated!

**Comment by alexellis** [comment]
LGTM @yankeexe 

@Yannig please see how we did this without changing the code of arkade itself.

Let me know if you see any differences in results.

**Pull request merged**